core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.html

```diff
@@ -39,6 +39,7 @@
                       !canWindowShowTwoCards() && !isCurrentSupplementalCardNonempty()}">
       <tutor-card displayed-card="displayedCard"
                   is-learn-again-button="isLearnAgainButton()"
+                  is-avatar-image-shown = "true"
```

Comment:
Follow existing style with regards to spacing around "=".

Reply:
Done